enterprise-suite/scripts/lbc.py

```diff
@@ -653,6 +683,8 @@ def main(argv):
     global args
     args = setup_args(argv)
 
+    check_new_install_script()
```

Comment:
This should be conditional on `skip_checks` flag, or it will brake lbc_tests.py otherwise.